### DISCUSSION TITLE
push 전 테스트 자동화

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#51] [New Feature]
일단 깃헙 액션이라도 넣어두면 어떨지 싶습니다

**Comment by janeljs** [comment]
좋습니다👍

**Comment by Dae-Hwa** [comment]
깃헙 액션 추가했습니다.
auth.key.properties는 github secrete에서 관리하도록 했습니다.

main브랜치에 push 혹은 pr 날리면 액션 작동해서 build작업을 거칩니다.

**Reply by janeljs** [reply]
고생하셨습니다! 완전 빠르게 적용해주셨네요ㅎㅎ
